docs/src/main/asciidoc/security-jwt-build.adoc

```diff
@@ -286,6 +313,7 @@ SmallRye JWT supports the following properties which can be used to customize th
 |smallrye.jwt.encrypt.key.id|`none`|Encryption key identifier which is checked only when JWK keys are used.
 |smallrye.jwt.new-token.signature-algorithm|RS256|Signature algorithm. This property will be checked if the JWT signature builder has not already set the signature algorithm.
 |smallrye.jwt.new-token.key-encryption-algorithm|RSA-OAEP|Key encryption algorithm. This property will be checked if the JWT encryption builder has not already set the key encryption algorithm.
+|smallrye.jwt.new-token.content-encryption-algorithm|A256GCM|Content encryption algorithm. This property will be checked if the JWT encryption builder has not already set the content encryption algorithm.
```

Comment:
I think I might have asked this before, but can't we include the generated documentation file instead?

Reply:
@gastaldi thanks, I've just updated it with 2 more properties (`sign.key`, `encrypt.key`).

Not sure it is possible as these are not in the Quarkus namespace yet - I'm just copying it from the smallrye-jwt docs for now. I'll need to work on making them visible as `quarkus.*` properties.

Does it answer your question ?

Reply:
Sure, that sounds good. Good job!

Reply:
Thanks @gastaldi